client/components/domains/connect-domain-step/index.jsx

```diff
@@ -160,11 +193,20 @@ function ConnectDomainStep( { domain, selectedSite, initialSetupInfo, initialSte
 
 ConnectDomainStep.propTypes = {
 	domain: PropTypes.string.isRequired,
+	domains: PropTypes.array,
 	selectedSite: PropTypes.object,
 	initialStep: PropTypes.string,
 	showErrors: PropTypes.bool,
+	hasSiteDomainsLoaded: PropTypes.bool,
 };
 
-export default connect( ( state ) => ( { selectedSite: getSelectedSite( state ) } ) )(
-	ConnectDomainStep
-);
+export default connect( ( state ) => {
+	const selectedSite = getSelectedSite( state );
+	const siteId = get( selectedSite, 'ID', null );
```

Comment:
There's an ongoing effort to deprecate `lodash` in Calypso (p4TIVU-9GZ-p2), so let's try to avoid using it 🙂 this could be replaced by `const siteId = selectedSite?.ID`

Reply:
Thanks, good catch! 😉